sessions/http-api-docs-that-work.md

```diff
@@ -0,0 +1,17 @@
+# Building HTTP API Docs that Work
+
+The HTTP API is arguably our most important API — the primary recommended use of IPFS is for people to run a daemon in one process and control it via the HTTP API in another — so it needs to be well documented. Let’s figure out how to fix that.
```

Comment:
There is already some context here that will be helpful for people to go through before jumping into this session. I'm sure you have more issues but here is my pick https://github.com/ipfs/http-api-spec/issues/116

Reply:
Hmmmm, I think there are a lot of good contextual points to bring up, but I’m also not trying to address most of the things in ipfs/http-api-spec#116 here. That issue is mostly about verifiability/validation (which is definitely important; I’m not disputing that), and my concern here is that the docs even say what they are trying to say clearly in the first place (regardless of whether it is up to date).

I’m hoping for this session to introduce existing problems (step 1) and collaboratively work through prototyping design solutions (what do the docs need to say to be useful? how does that info need to be arranged to be clear?) (step 2) as the main part of this — setting up understanding among a lot of people (especially since I won’t be around anymore) of what these docs need to be in order to work for IPFS’s users (and potential users). I’ll add some more on step 3/goal 3 below, where you called it out specifically.